.eslintrc.json

```diff
@@ -9,7 +9,11 @@
     "sourceType": "module"
   },
   "rules": {
+<<<<<<< HEAD
     "no-console": "off",
+=======
+    "no-console": "off"
+>>>>>>> 91afe82 (feat(authentication):signin)
```

Comment:
Great work Egide, Try to resolve these conflicts

Reply:
Thanks